src/mpcCoreKit.ts

```diff
@@ -196,8 +196,14 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     return this.options.uxMode === UX_MODE.REDIRECT;
   }
 
+  // use dkg is always false for ed25519 key type and for secp256k1 key type when options.useDKG is false
   private get useDKG(): boolean {
-    return this.keyType === KeyType.ed25519 && this.options.useDKG === undefined ? false : this.options.useDKG;
+    return this.keyType === KeyType.ed25519 ? false : this.options.useDKG;
+  }
+
+  // import flow is used when this.options.useDKG is undefined or false
```

Comment:
if `useDKG` is undefined, this errors? can `useDKG` actually be undefined?

Reply:
if user doesnt pass anything explicity in options.useDKG thn it will be undefined.

Reply:
then the function signature `private get useDKG(): boolean` should be adjusted